math/wide_integer/uintwide_t.h

```diff
@@ -3750,6 +3753,8 @@
   using  int4096_t  = uintwide_t< 4096U, std::uint32_t, void, true>;
   using  int8192_t  = uintwide_t< 8192U, std::uint32_t, void, true>;
   using  int16384_t = uintwide_t<16384U, std::uint32_t, void, true>;
+  using  int32768_t = uintwide_t<32768U, std::uint32_t, void, true>;
+  using  int65536_t = uintwide_t<65536U, std::uint32_t, void, true>;
 
   #if !defined(WIDE_INTEGER_DISABLE_TRIVIAL_COPY_AND_STD_LAYOUT_CHECKS)
   static_assert(std::is_trivially_copyable<int64_t   >::value && std::is_standard_layout<int64_t   >::value, "uintwide_t must be trivially copyable with standard layout.");
```

Comment:
Do these tests not belong in test code?

Reply:
What do you mean exactly, John. Maybe I made a mistake. I had attempted to create convenience types `(u)int32768_t` and `(u)int65536_t`.

The intent was to add trivial test cases for these in the spot tests [here](https://github.com/ckormanyos/wide-integer/blob/9c06b04701edf849a1c8ffae231cbae845ce7ec8/test/test_uintwide_t_spot_values.cpp#L47)